mhkit/wave/io/wave_hindcast.py

```diff
@@ -0,0 +1,84 @@
+        hsconfigure
+        and enter at the prompt:
+        hs_endpoint = https://developer.nrel.gov/api/hsds
+        hs_username = None
+        hs_password = None
+        hs_api_key = {your key}
+        Parameters
+        ----------
+        wave_path : string
+            Path to US_Wave .h5 files
+            Available formats:
+                /nrel/US_wave/US_wave$_{year}.h5
+                /nrel/US_wave/US_wave$_*.h5 (Only use when specifying years parameter)
+        parameter: string
+            dataset parameter to be downloaded
+            spatial dataset options: directionality_coefficient, energy_period, maximum_energy_direction
+                mean_absolute_period, mean_zero-crossing_period, omni-directional_wave_power, peak_period
+                significant_wave_height, spectral_width, water_depth 
+        lat_lon: tuple or list of tuples
+            latitude longitude pairs at which to extract data 
+        years : list, optional
+            List of years to access, by default None
+        tree : str | cKDTree (optional)
+            cKDTree or path to .pkl file containing pre-computed tree
+            of lat, lon coordinates
+        unscale : bool (optional)
+            Boolean flag to automatically unscale variables on extraction
+        str_decode : bool (optional)
+            Boolean flag to decode the bytestring meta data into normal
+            strings. Setting this to False will speed up the meta data read.
+        hsds : bool (optional)
+            Boolean flag to use h5pyd to handle .h5 'files' hosted on AWS
+            behind HSDS
+
+        Returns
+        ---------
+        data: pandas DataFrame 
+            Data indexed by datetime with columns named for parameter and lat_lon 
+        meta: pandas DataFrame 
+            location metadata for the requested data location   
+        """
+        
+        assert isinstance(parameter, str), 'parameter must be of type string'
+        assert isinstance(lat_lon, (list,tuple)), 'lat_lon must be of type list or tuple'
+
+        
+        
+        
+        if years != None or '*' in wave_path:
+            waveKwargs = {'tree':tree,'unscale':unscale,'str_decode':str_decode, 'hsds':hsds,
+            'years':years}
+            rex_accessor = MultiYearWaveX
+        else:
+            waveKwargs = {'tree':tree,'unscale':unscale,'str_decode':str_decode, 'hsds':hsds}
+            rex_accessor = WaveX
+        names = []    
+        with rex_accessor(wave_path, **waveKwargs) as waves:
+            data = waves.get_lat_lon_df(parameter,lat_lon)
+            col = data.columns[:]
+            meta = waves.meta.loc[col,:]
+            
+            if isinstance(lat_lon[0], (list,tuple)):
+                for c,l in zip(col,lat_lon):
+                    temp = parameter+'_'+str(l[0])+'_'+str(l[1])
+                    data = data.rename(columns={c:temp})
+            else:
+                temp = parameter+'_'+str(lat_lon[0])+'_'+str(lat_lon[1])
```

Comment:
suggest f-string again:
`temp = f'{parameter}_{lat_lon[0]}_{lat_lon[1]}'`

Reply:
addressed in https://github.com/MHKiT-Software/MHKiT-Python/pull/60/commits/151327767fd268ae8b9cae9beda9f68b84cd23b1